erigon-lib/state/domain.go

```diff
@@ -1281,9 +1256,6 @@ func (d *Domain) BuildMissedAccessors(ctx context.Context, g *errgroup.Group, ps
 		})
 	}
 	for _, item := range d.missedAccessors() {
-		if d.IndexList&AccessorHashMap == 0 {
```

Comment:
are you sure on removing this section? not all domains have same accessor types.

Reply:
I moved this check to `d.missedAccessors()`  so that it return nil if IndexList doesn't have accessorHashMap. So the only change is that the check is done once at "domain level", and not for each file like before. Which is okay? Apart from this they're functionally equivalent. Or am I missing something?